modules/nw-egress-router-about.adoc

```diff
@@ -41,6 +62,20 @@ The environment variables determine which addresses the egress-router image uses
 Network Address Translation (NAT) rules are set up so that connections to the cluster IP address of the pod on any TCP or UDP port are redirected to the same port on IP address specified by the `EGRESS_DESTINATION` variable.
 
 If only some of the nodes in your cluster are capable of claiming the specified source IP address and using the specified gateway, you can specify a `nodeName` or `nodeSelector` indicating which nodes are acceptable.
+endif::openshift-sdn[]
+
+ifdef::ovn[]
+An egress router is a pod that has two network interfaces.
+For example, the pod can have `eth0` and `net1` network interfaces.
+The `eth0` interface is on the default cluster network and the pod
+continues to use the interface for ordinary cluster-related network traffic.
+The `net1` interface has an IP address and gateway from the external physical network.
+Other pods in the {product-title} cluster can access the egress router service and the service enables the pods to access external services.
+The egress router acts as a bridge between pods and an external system.
+
+Traffic that leaves the egress router exits through a node, but the packets
```

Comment:
I'm always torn as to how persistent to be regarding the fact that it's a _pod_. It could be if firmly established earlier that this implementation uses a pod to redirect the traffic, all subsequent occurrences can simply be _egress router_ and it's understood what that means in terms of the feature architecture.

So it's probably okay as is.